calico-cloud/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Calico Cloud
+description: Learn about Calico Cloud.
+canonical_url: '/calico-cloud/index'
+---
+
+![calico-cloud]({{site.baseurl}}/images/calico-cloud-small.png)
+
+## If you could solve these 3 problems today...
+
+#### Workload access controls
+
+Implement pod-level workload access controls and protect containerized environments in the Kubernetes cluster from outside threats, while enabling applications and workloads to securely communicate with resources outside the cluster behind a firewall or other control point.
+
+#### Compliance
+
+Create compliance policies for any compliance framework including PCI, SOC 2, and more. Continuously monitor compliance for your container, Kubernetes, and cloud environment. Retain a daily history of your compliance status. Generate on-demand or scheduled audit reports.
+
+#### Faster troubleshooting
+
+Enable faster troubleshooting of Kubernetes workloads and applications with Service Graph, Packet Capture, anomaly detection, and performance hotspots, leading to shorter time-to-resolution, less application downtime, and improved quality of service
+
+## then take your cluster into Calico Cloud and see for yourself!
+
+### Don't have a Calico cluster? Easy...[Try it now](https://www.tigera.io/tigera-products/cloud-trial)
```

Comment:
The semaphore error is because you have a html link inside a header, and headers become link themselves on the right navbar.

Either remove the link from this header, or disable the right navbar for this page.

Reply:
Fixed